fix(wexond) circumvent security measures from Google

lmancel opened the pull request:
- alter user agent for most requests
- do not unnecessarily alter request and response
- ensure most background url are not undefined sometimes favicons do not exist -> do not use "undefined" as value...
- refactor: better url and title update
- chore: View does not extend BrowserView anymore https://github.com/electron/electron/issues/23#issuecomment-19613241
- chore: some typing/linting

Commit:
fix(wexond) circumvent security measures from Google

- alter user agent for most requests
- do not unnecessarily alter request and response
- ensure most background url are not undefined
  sometimes favicons do not exist -> do not use "undefined" as value...
- refactor: better url and title update
- chore: View does not extend BrowserView anymore
  https://github.com/electron/electron/issues/23#issuecomment-19613241
- chore: some typing/linting

Commit:
misc: avoid copying electron version in several places

Having to change the version in several places is prone to errors
Now it's enough to only update the electron dependency
To be validated on Windows

Azevedo-Dani commented on src/wexond/renderer/app/components/Bubble/index.tsx:
```diff
@@ -25,18 +27,19 @@ export const Bubble = ({
-  toggled?: boolean;
-}) => {
-  const width = getSize(itemsPerRow);
+  children?: any
```
```suggestion
  children?: JSX.Element
```

OR 
```suggestion
 export const Bubble = React.FC<{icon?: string
  invert?: boolean
  maxLines?: number
  iconSize?: number
  onClick?: (e?: React.MouseEvent<HTMLDivElement>) => void
  onCloseClick?: (e?: React.MouseEvent<HTMLDivElement>) => void
  itemsPerRow?: number
  disabled?: boolean
  toggled?: boolean}> = ....
```

lmancel replied:
React.FC does not seem to be available in this version

Azevedo-Dani commented on src/wexond/renderer/app/models/tab.ts:
```diff
@@ -144,48 +144,45 @@ export class Tab {
       },
     )
 
+    ipcRenderer.on(`view-title-updated-${this.id}`, (e: any, title: string) => {
```
Seems not used
```suggestion
    ipcRenderer.on(`view-title-updated-${this.id}`, (_, title: string) => {
```

Azevedo-Dani reviewed (approved):
just some remarks related to types. not mandatory
LGTM

Azevedo-Dani commented on src/wexond/renderer/app/components/Bubble/index.tsx:
```diff
@@ -1,18 +1,20 @@
+import ensureIcon from '../../utils/ensureIcon'
+
+export const getSize = (i: number): number => {
+  const width = 800
```
why 800 there's a reason ?

Azevedo-Dani replied:
I saw it was like that before my bad

Commit:
fix(keyboard) parent of keyboard should be focused

do not modify keyboard state if parent window is not focused
this prevents state changes when the keyboard receives the focus

CHRUL-413

Commit:
fix(wexond) set parent on opened windows

prevent child windows going behind the main window

CHRUL-421

Commit:
misc(tab) use "_" instead of unused function param

Commit:
misc(Bubble) any -> JSX.Element
